.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1.1.2
+      with:
+        ruby-version: 2.6
+    
+    - name: Install dependencies
+      run: bundle install
+    
+    - name: Create package
+      run: bundle exec rake package
+    
+    - name: finding packages...
+      run: |
+        cd /github/workspace/packaging/builds/
+        echo ::set-env name=version::$(ls)
+        echo ::set-env name=deb_file::$(find . -type f -name "*.deb")
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: v${{ env.version }}
+        release_name: Release ${{ env.version }}
+        body: |
+          Changes in this Release
```

Comment:
I think we can use a generic message (see https://github.com/Shopify/shopify-app-cli/releases) so we won't need to worry about identifying the particular changes.